### PR TITLE
feat(cico): use add funds crypto exchange experiment params

### PR DESCRIPTION
**Pull request by satish-ravi**
### Description

Actually uses the experiment params and implements the experiment variant to use the appropriate text and navigate to the right screen.

https://user-images.githubusercontent.com/5062591/207996690-6e1d6836-7a74-47ca-a740-d0d80bfbeb2a.mp4

### Other changes

Renamed enums to exactly match screen names

### Tested

Unit, manual by setting up overrides in statsig

### How others should test

1. Add your wallet address to the appropriate variant of the [experiment in statsig](https://console.statsig.com/4plizaPmWwPL21ASV4QAO0/experiments/add_funds_crypto_exchange_qr_code/setup)
2. Navigate to Add funds, select any currency and enter any amount
3. Ensure external exchanges section is visible on the "Select Payment Method" screen with the appropriate text ("Deposit From" for variant, "Cryptocurrency Exchange" for control)
4. Click on exchanges section and ensure navigation to correct screen ( new Deposit Crypto QR code screen for variant, existing external changes screen for control)

### Related issues

- Fixes ACT-544

### Backwards compatibility

Yes

### CODE REVIEW

File: src/fiatExchanges/SelectProvider.test.tsx [hunk 1]
```diff
@@ -322,36 +420,3 @@ describe(SelectProviderScreen, () => {
     })
   })
 })
-
-describe('getAddFundsCryptoExchangeExperimentParams', () => {
```
Comment (satish-ravi) [inline src/fiatExchanges/SelectProvider.test.tsx:326]: these are now covered directly in the exchanges section text

File: src/fiatExchanges/types.ts [hunk 2]
```diff
@@ -4,6 +4,6 @@ export enum SelectProviderExchangesText {
 }
 
 export enum SelectProviderExchangesLink {
-  ExchangesScreen = 'ExchangesScreen',
-  ExchangesQrScreen = 'ExchangesQrScreen',
+  ExternalExchangesScreen = 'ExternalExchangesScreen',
```
Comment (jophish) [inline src/fiatExchanges/types.ts:7]: nit: kind of strange that these have different naming conventions, but i see they match what's in statsig/line up with our screen names.

**Review by jophish — APPROVED** [review]
LGTM :slightly_smiling_face:

File: locales/base/translation.json [hunk 3]
```diff
@@ -1281,6 +1281,8 @@
     "cryptoExchange": "Cryptocurrency Exchange",
     "feesVary": "Fees Vary",
     "viewExchanges": "View Exchanges",
+    "depositFrom": "Deposit From",
```
Comment (jh2oman) [inline locales/base/translation.json:1284]: nit: designs show `Deposit from` with a lowercase f.

**Review by jh2oman — APPROVED** [review]

File: locales/base/translation.json (hunk 3 above)
Comment (satish-ravi) [reply locales/base/translation.json]: good catch. fixed

**Pull request merged**